Code/Tools/ProjectManager/Source/AddRemoteTemplateDialog.cpp

```diff
@@ -48,6 +48,8 @@ namespace O3DE::ProjectManager
 
         m_repoPath = new FormLineEditWidget(tr("Remote URL"), "", this);
         m_repoPath->setMinimumSize(QSize(600, 0));
+        m_repoPath->setErrorLabelText(tr("Not a valid remote template source."));
+        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/o3de/example.git");
```

Comment:
lets use `https` instead of `http` 
```suggestion
        m_repoPath->lineEdit()->setPlaceholderText("https://github.com/o3de/example.git");
```